### DISCUSSION TITLE
Combine updates of shared modules in a monorepo, currently we have multiple PRs for each package in monorepo

### DISCUSSION THREAD

**Discussion by hitesh-parmar** [renovatebot/renovate#20699] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### If you're self-hosting Renovate, tell us what version of the platform you run.

GitHub (.com and Enterprise Server)

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hello Everyone,

I am using renovate in a monorepo which consists of 2 packages A and B. Both these packages have exact same dependency to node module C. 
When new version of C is available, I get 2 PRs one for update of dependency in module A and one PR for B.

I would like to have one PR in such a case. 

### What you tried so far.

I couldn't find this in documentation.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment]
use package rules to group those deps

**Reply by rarkins** [reply]
It's the same dependency so I'm guessing they've configured additionalBranchPrefix

**Reply by hitesh-parmar** [reply]
Thanks @rarkins . Yes you are right, we are using additionalBranchPrefix and this was the reason separate PRs were created.

**Comment by rarkins** [comment, marked as answer]
What are the two branch names you get?

**Reply by hitesh-parmar** [reply]
Solved with https://github.com/renovatebot/renovate/discussions/20699#discussioncomment-5189211
